### PR TITLE
GitHub action resurrected

### PR DESCRIPTION
**Pull request by p5k369**
Removed pre-compiled libraries, temporary commented tests that are stopping test pipeline.

2 Issues are stopping the github test pipeline.
- All tests that depend on DisplayCAL.config.initcfg fail because of a scenario that might be something like: A temporary directory is created, than changed dir into it, than the directory gets deleted , the directory is not changed and os.cwd() gets called
- test_enumerate_displays fails because Xvfb cannot provide edid information. If this is relevant for further tests than this might be a showstopper for Xvfb.

### CODE REVIEW

**Comment by eoyilmaz** [comment]
Perhaps we need to mock ``RealDisplaySizeMM`` functions to return known values, so the other parts work properly. Would you like to do it?

**Comment by eoyilmaz** [comment]
https://github.com/eoyilmaz/displaycal-py3/issues/36

**Comment by p5k369** [comment]
The tests depending on RealDisplaySizeMM are working but the problem with test_get_default_size_1 still exists and I cant figure out what the problem is.
Compared to others the function under test (get_default_size()) is relatively short  and 
uncomplex. I just cant find the connection to the FileNotFoundError.

File: DisplayCAL/RealDisplaySizeMM.py [hunk 1]
```diff
@@ -129,8 +128,7 @@ def get_display(display_no=0):
         if argyll_display.endswith(" [PRIMARY]"):
             argyll_display = " ".join(argyll_display.split(" ")[:-1])
         for display in _displays:
-            desc = display.get("description")
-            if desc:
+            if desc := display["description"]:
```
Comment (eoyilmaz) [inline DisplayCAL/RealDisplaySizeMM.py:131]: cool we have the walrus operator :+1: 
we are officially supporting only Python 3.8+ then.

**Comment by eoyilmaz** [comment]
Cool stuff, thanks for all of your efforts, I would like to say my appreciation with a Turkish expression "Ellerine saglik!" (roughly translates to "health to your hands for your great work"). There are a couple of questions but the rest seems okay. Thank you again.

**Comment by p5k369** [comment]
I think you are the one who has to be thanked because of bringing DisplayCal rolling and back to life.

**Comment by p5k369** [comment]
Finally. I got test_get_default_size_1 up and running again. Take a look at #41 
However I wouldn't count too much on the coverage for this specific function as there is some amount of code that cant be reached.
Stuff like:
```
...
   ppi_def = get_default_dpi()
   method = 1
       if method == 0:
          # use configurable screen diagonal
          inch = 20.0
          mm = inch * 25.4
...
```

**Pull request merged**